bitnami/airflow/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: airflow
-version: 5.0.6
+version: 5.0.7
```

Comment:
This seems to implement a new feature, so I guess a minor release would be more appropriate (5.1.0). Could you change it?

Reply:
version upgraded to 5.1.0